CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
+* **sources:** CSS Component Image; deprecation img-rounded, img-thumbnail in fractal ([#1188](https://github.com/dso-toolkit/dso-toolkit/issues/1188))
```

Comment:
Graag een inhoudelijk kloppende titel voor het issue opstellen en die dupliceren in de CHANGELOG.md.

("in fractal" is een detail wat niet interessant is voor de buitenwereld en ook niet waar: deprecation is dso-toolkit eigen)